src/jit/hwintrinsicArm64.h

```diff
@@ -54,6 +54,16 @@ struct HWIntrinsicInfo
     static const HWIntrinsicInfo& lookup(NamedIntrinsic id);
     static int lookupNumArgs(const GenTreeHWIntrinsic* node);
 
+    static bool isFullyImplementedIsa(InstructionSet isa);
+    static bool isScalarIsa(InstructionSet isa);
+
+    // Member lookup
+
+    static NamedIntrinsic lookupId(NamedIntrinsic id)
```

Comment:
This looks strange, what is this used for?

Reply:
I think for dumping but I'm not actually sure. I just copied it over from the x86 implementation with the others